common/data_refinery_common/models/models.py

```diff
@@ -205,6 +205,17 @@ def save(self, *args, **kwargs):
         return super(SampleAnnotation, self).save(*args, **kwargs)
 
 
+class ProcessedPublicObjectsManager(models.Manager):
+    """
+    Only returns Experiments that are is_public and have related is_processed Samples.
+    """
+    def get_queryset(self):
+        return super().get_queryset().filter(
+            is_public=True, 
+            samples__is_processed=True, 
```

Comment:
Is this true if there exists at least one processed sample or if all samples are processed?